### PR TITLE
Update german.txt

### PR DESCRIPTION
**Pull request by danidoedel**

### CODE REVIEW

File: data/language/german.txt [hunk 1]
```diff
@@ -2720,24 +2720,24 @@ STR_2714    :-
...
-STR_2727    :???
-STR_2728    :???
-STR_2729    :???
-STR_2730    :???
-STR_2731    :???
-STR_2732    :???
-STR_2733    :???
-STR_2734    :???
-STR_2735    :???
+STR_2718    :(hoch)
```
Comment (Gymnasiast) [inline data/language/german.txt]: The original string meant 'go up one directory'. Is this the correct German translation for that?

File: data/language/german.txt [hunk 2]
```diff
@@ -2753,22 +2753,22 @@ STR_2747    :{ENDQUOTES}
...
 # New strings used in the cheats window previously these were ???
-STR_2750    :???
-STR_2751    :???
+STR_2750    :Alle Gegenstände nach oben bewegen
+STR_2751    :Alle Gegenstände nach unten bewegen
 STR_2752    :Gras beseitigen
 STR_2753    :Rasen mähen
 STR_2754    :Pflanzen gießen
-STR_2755    :Vandalismus beseitigen
+STR_2755    :{SMALLFONT}Vandalismus beseitigen
```
Comment (Gymnasiast) [inline data/language/german.txt]: Should not contain {SMALLFONT}

File: data/language/german.txt [hunk 3]
```diff
@@ -2782,21 +2782,21 @@ STR_2775    :Vollbild (Desktop)
 STR_2776    :Sprache
 STR_2777    :{MOVE_X}{SMALLFONT}{STRING}
 STR_2778    :{RIGHTGUILLEMET}{MOVE_X}{SMALLFONT}{STRING}
-STR_2779    :???
-STR_2780    :???
+STR_2779    :Ansichtsfenster #{COMMA16}
```
Comment (Gymnasiast) [inline data/language/german.txt]: Will shows up 'Ansichtsfenster #1' and such, which I doubt is valid in German. You should probably drop the hash (#).

**Comment by Gymnasiast** [comment]
Thanks for giving the German translation a much needed improvement.
It's looking good, but I found three strings you should take a look at. I put them as line notes.

File: data/language/german.txt (hunk 1 above)
Comment (danidoedel) [inline data/language/german.txt]: Yes, the word "Hoch" is also used by Windows Explorer in this context, but I changed it to upper case.
hoch

File: data/language/german.txt (hunk 2 above)
Comment (danidoedel) [inline data/language/german.txt]: Ok I fixed that.

File: data/language/german.txt (hunk 3 above)
Comment (danidoedel) [inline data/language/german.txt]: You're right, I fixed that.

**Comment by danidoedel** [comment]
I fixed these three strings. 
No Problem, I'm glad to help with the translation.

**Comment by Gymnasiast** [comment]
Thanks! The changes will be synchronised at 05:00, so they'll be in the game tomorrow.

**Pull request merged**